### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by michaelajr**
Removed compatibility with Python SDK - left Java SDK - and linked to compatibility section.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -186,6 +183,11 @@ Compatability with the AWS Encryption SDK
 Encryption SDK.** But not all files encrypted with the AWS Encryption
```
Comment (mattsb42-aws) [inline README.rst]: Would you mind updating this statement to specify the Encryption SDK for Java as well? I don't want to send mixed messages, and technically "the AWS Encryption SDK" is the standard, not any specific client.

**Review by mattsb42-aws — COMMENTED** [review]
Thanks! This addresses my concerns with the intro.

**Comment by michaelajr** [comment]
@mattsb42-aws for your review. I now only reference compatibility the the Java SDK and renamed the compatibly section to be Java-specific. Also updated all references to the "SDK" to be the "Java SDK".

**Comment by mattsb42-aws** [comment]
Looks good, thanks!

**Pull request merged**